client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/content/get-domain-footer-details.ts

```diff
@@ -0,0 +1,34 @@
+import { translate } from 'i18n-calypso';
+import ThankYouFooterDetailProps from 'calypso/components/thank-you-v2/footer';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+export default function getDomainFooterDetails( limit?: number ): ThankYouFooterDetailProps[] {
+	const details = [
+		{
+			key: 'footer-domain-essentials',
+			title: translate( 'Dive into domain essentials' ),
+			description: translate(
+				'Check out our support documentation for step-by-step instructions and expert guidance on your domain set up.'
+			),
+			buttonText: translate( 'Master the domain basics' ),
+			buttonHref: '/support/domains',
+			buttonOnClick: () => {
+				recordTracksEvent( 'calypso_thank_you_footer-domain-essentials' );
+			},
+		},
+		{
+			key: 'footer-domain-resources',
+			title: translate( 'Your go-to domain resource' ),
+			description: translate(
+				'Dive into our comprehensive support documentation to learn the basics of domains, from registration to management.'
+			),
+			buttonText: translate( 'Domain support resources' ),
+			buttonHref: '/support/category/domains-and-email/',
+			buttonOnClick: () => {
+				recordTracksEvent( 'calypso_thank_you_footer-domain-resources' );
+			},
+		},
+	];
+
+	return details.slice( 0, limit ?? details.length );
+}
```

Comment:
I'm not entirely sure that this file should be located under `/redesign-v2/pages/content` mostly because we already have a `/redesign-v2/products` folder - which could be considered as content as well (said different I would have seen `content` and `products` at the same level). However, it's probably better to wait until we've refactored more `Thank You` pages to find the perfect location.

Reply:
Yeah, that's a good point! Let's keep that in mind as we continue the refactoring!